packages/netlify-cms-widget-markdown/src/serializers/index.js

```diff
@@ -115,10 +115,11 @@ export const remarkToMarkdown = obj => {
     listItemIndent: '1',
 
     /**
-     * Settings to emulate the defaults from the Prosemirror editor, not
-     * necessarily optimal. Should eventually be configurable.
+     * Use asterisk for everything, it's the most versatile. Eventually using
+     * other characters should be an option.
      */
     bullet: '*',
+    emphasis: '*',
```

Comment:
Don't use underscores at all. This fixes some major parsing issues while introducing some edge cases, fair trade for now. The latest Slate, 0.52+, has a completely new model that mirrors MDAST, the model we're parsing Slate documents into. This should help address a whole lot of parsing issues and bring us very close to Commonmark compliance.